BUILD.md

```diff
@@ -146,33 +176,32 @@ curl https://run.linkerd.io/emojivoto.yml | bin/linkerd inject - | kubectl apply
 kubectl -n emojivoto port-forward svc/web-svc 8080:80
 
 # view details per deployment
-bin/linkerd -n emojivoto stat deployments
+bin/linkerd viz -n emojivoto stat deployments
 
 # view a live pipeline of requests
-bin/linkerd -n emojivoto tap deploy voting
+bin/linkerd viz -n emojivoto tap deploy voting
 ```
 
 #### Deploying Control Plane components with Tracing
 
 Control Plane components have the `trace-collector` flag used to enable
 [Distributed Tracing](https://opentracing.io/docs/overview/what-is-tracing/) for
 development purposes. It can be enabled globally i.e Control plane components
-and their proxies by using the `--control-plane-tracing` installation flag.
+and their proxies by using the `--set controlPlaneTracing=true` installation
+flag.
 
 This will configure all the components to send the traces at
 `linkerd-collector.{{.Namespace}}.svc.{{.ClusterDomain}}:55678`
 
 ```bash
 
 # install Linkerd with tracing
-linkerd install --control-plane-tracing | kubectl apply -f -
+linkerd install --set controlPlaneTracing=true | kubectl apply -f -
 
-# install OpenCensus collector and Jaeger collector to collect traces
-linkerd inject https://gist.githubusercontent.com/Pothulapati/245842ce7f319e8bcd02521460684d6f/raw/52c869c58b07b17caeed520aa91380c2230d6e0c/linkerd-tracing.yaml --manual | kubectl apply -f -
+# install the Jaeger extension
+linkerd jaeger install | kubectl apply -f -
 ```
```

Comment:
perhaps add a note that control plane components need to be restarted after the jaeger-injector is deployed before they will be able to export traces.

Reply:
:+1: